en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md

```diff
@@ -14,13 +14,65 @@ There are two ways to add an API to the Choreo Connect.
 
 ### Step 1 - Configure Choreo Connect with API Manager
 
-Please refer [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/#configure-choreo-connect-with-api-manager).
+!!! info
+    **Before you begin**
+
+    - Make sure you have installed Docker and Docker Compose on your machine.
+
+    - Download the latest [Choreo Connect release](https://github.com/wso2/product-microgateway/releases) and extract it to a folder of your choice. The extracted folder will be referred to as `CHOREO-CONNECT_HOME` here onwards.
+
+    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps [here](https://github.com/wso2/product-apim#installation--running).
+
+### Step 1 - Find the API-M IP Address
+
+Choreo Connect identifies the location of the API-M server instance by its IP address. If you are trying out WSO2 API Manager locally, you can use the private IP retrieved using `hostname -I` or `ipconfig`.
+
+### Step 2 - Update the Choreo Connect Config File
+
+Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update the `[controlPlane]` configuration as explained below:
```

Comment:
we should use the choreo-connect-with-apim folder

Reply:
I added the link to the quick start guide of cc with apim docker compose.